kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -201,16 +201,20 @@ prove cindex = do
 
 showGraph
     :: MonadIO m
+    => MonadWriter String m
     => Maybe FilePath
     -> MonadState (ReplState claim) m
     => m ()
 showGraph mfile = do
     graph <- getInnerGraph
     axioms <- Lens.use lensAxioms
-    liftIO $ maybe
-            (showDotGraph (length axioms) graph)
-            (saveDotGraph (length axioms) graph)
-            mfile
+    installed <- liftIO Graph.isGraphvizInstalled
+    case installed of
```

Comment:
I would probably use `if ... then ... else` instead of `case ... of ...` for a `Bool`.

Reply:
Yeah, you're right. Thanks!